### PR TITLE
Fixes to compose syntax for `volumes_from`, `volumes`, and optional `cpu` parameter

### PR DESCRIPTION
**Pull request by philipn**
- The `cpu` parameter in ECS is now optional.  I think it was required before.  So we don't emit an error message when it's not present.
- Allow the compose v2 style 'service:name:ro' syntax in `volumes_from` clauses.
- Allow the compose style ':rw' in `volumes` clauses.
- Emit `readOnly` on ECS `volumesFrom` clauses, which were previously skipped.

### CODE REVIEW

**Comment by cjerdonek** [comment]
Love it! :+1:

File: container_transform/compose.py [hunk 1]
```diff
@@ -218,7 +218,36 @@ def emit_entrypoint(self, entrypoint):
...
 
     def ingest_volumes_from(self, volumes_from):
-        return volumes_from
+        ingested_volumes_from = []
+        for vol in volumes_from:
+            ingested = {}
+            parts = vol.split(':')
+            rwo_value = None
+
+            assert(len(parts) <= 3)
```
Comment (micahhausler) [inline container_transform/compose.py]: Could you add an explanation parameter to this assert so the error is a little more clear? ``assert(len(parts) <=3, "volume string '{}' has too many colons".format(vol))` or something like that?

**Comment by micahhausler** [comment]
This is great! Thanks for submitting. Just one comment on the compose `ingest_volumes_from`, and we're good to go.

**Comment by micahhausler** [comment]
@philipn Any chance you could fix the indentation for the failing CI?

**Comment by sveneh** [comment]
any blockers for this to be merged?

**Pull request merged**